packages/suite/src/components/suite/Preloader/SuiteLayout/DeviceSelector/DeviceSelector.tsx

```diff
@@ -155,6 +155,12 @@ export const DeviceSelector = () => {
             }),
         );
 
+    const handleRefreshClick = () => {
+        if (deviceNeedsRefresh) {
+            dispatch(acquireDevice(selectedDevice));
+        }
+    };
```

Comment:
Currently this is unreachable because the `DeviceSelector` receives the click event :( 

The hover effect of the selector should be disabled when hovering over the refresh button, as well as its `onClick` functionality, and I would at least change opacity of the "Refresh" text to indicate it's clickable.

https://github.com/trezor/trezor-suite/assets/45338719/cef64fdb-8300-435e-a4c7-afab8c4cf9ac

Reply:
2723fb25a12f3e47d1ef585f7e50cb41092b6ee7